### PR TITLE
Update us.m3u

### PR DESCRIPTION
**Pull request by Tribalize**
Updating of dead links added some categories while fixing channels. Tested channels as stated in Pull Request Guidelines

### CODE REVIEW

File: channels/us.m3u [hunk 1]
```diff
@@ -3445,75 +3445,75 @@ https://dai.google.com/linear/hls/event/Xu-I6qGiSTKeu6glzS4BtQ/master.m3u8
...
 #EXTINF:-1 tvg-id="TVSFamilyChannel.us" tvg-name="TVS Family Channel" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/iuQyJ7f.jpg" group-title="Family",TVS Family Channel (360p)
-https://nrpus.bozztv.com/36bay2/gusa-TVSFamilyChannel/index.m3u8
-#EXTINF:-1 tvg-id="TVSFilmNoirNetwork.us" tvg-name="TVS Film Noir Network" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/pXOyNEy.png" group-title="",TVS Film Noir Network (360p)
-https://nrpus.bozztv.com/36bay2/gusa-TVSFilmNoir/index.m3u8
-#EXTINF:-1 tvg-id="TVSFrontPageDetective.us" tvg-name="TVS Front Page Detective" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/pXOyNEy.png" group-title="",TVS Front Page Detective (360p)
-https://nrpus.bozztv.com/36bay2/gusa-tvsfrontpagedetective/index.m3u8
+https://rpn1.bozztv.com/36bay2/gusa-TVSFamilyChannel/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="TVSFilmNoirNetwork.us" tvg-name="TVS Film Noir Network" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/pXOyNEy.png" group-title="Movies",TVS Film Noir Network (360p)
+https://rpn1.bozztv.com/36bay2/gusa-TVSFilmNoir/tracks-v1a1/index-1623400693-now.m3u8
```
Comment (LaneSh4d0w) [inline channels/us.m3u:3462]: Better use this : https://rpn1.bozztv.com/36bay2/gusa-TVSFilmNoir/tracks-v1a1/mono.m3u8

File: channels/us.m3u [hunk 2]
```diff
@@ -3445,75 +3445,75 @@ https://dai.google.com/linear/hls/event/Xu-I6qGiSTKeu6glzS4BtQ/master.m3u8
...
+#EXTINF:-1 tvg-id="TVSFrontPageDetective.us" tvg-name="TVS Front Page Detective" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/pXOyNEy.png" group-title="Classic",TVS Front Page Detective (360p)
+https://rpn1.bozztv.com/36bay2/gusa-tvsfrontpagedetective/tracks-v1a1/index-1623400775-now.m3u8
```
Comment (LaneSh4d0w) [inline channels/us.m3u:3464]: Better use this :  https://rpn1.bozztv.com/36bay2/gusa-tvsfrontpagedetective/tracks-v1a1/mono.m3u8

File: channels/us.m3u [hunk 3]
```diff
@@ -3445,75 +3445,75 @@ https://dai.google.com/linear/hls/event/Xu-I6qGiSTKeu6glzS4BtQ/master.m3u8
...
-https://nrpus.bozztv.com/36bay2/gusa-TVSTodayHomeEntertainment/tracks-v1a1/index.m3u8
-#EXTINF:-1 tvg-id="TVSTravelNetwork.us" tvg-name="TVS Travel Network" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/Bgqy4fL.jpg" group-title="Travel",TVS Travel Network (720p)
-https://nrpus.bozztv.com/36bay2/gusa-tvstravel/index.m3u8
-#EXTINF:-1 tvg-id="TVSTurbo.us" tvg-name="TVS Turbo" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/RMewTGr.jpg" group-title="Classic",TVS Turbo (360p)
-https://nrpus.bozztv.com/36bay2/gusa-tvsturbo/index.m3u8
+#EXTINF:-1 tvg-id="TVSTodayHomeEntertainmentNetwork.us" tvg-name="TVS Today Home Entertainment Network" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/pXOyNEy.png" group-title="General",TVS Today Home Entertainment Network
+https://rpn1.bozztv.com/36bay2/gusa-TVSTodayHomeEntertainment/tracks-v1a1/index-1623403745-now.m3u8
+#EXTINF:-1 tvg-id="TVSTravelandFood" tvg-name="TVS Travel & Food" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/Bgqy4fL.jpg" group-title="Travel",TVS Travel & Food Network (720p)
```
Comment (Dum4G) [inline channels/us.m3u:3509]: Incorrect tvg-id

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/us.m3u (hunk 3 above)
Comment (Tribalize) [reply channels/us.m3u:3509]: you are correct. Old link was dead. I cannot find tvg-id for the new channel. can cancel the pull request. I will pull that channel

Comment (Dum4G) [reply channels/us.m3u:3509]: I mean you just didn't place .us in the end, no need to close the request

Comment (Tribalize) [reply channels/us.m3u:3509]: just did another pull. i will update that channel with the next pull. still going through the list

**Pull request closed without merging**